### DISCUSSION TITLE
Advanced bee farm template

### DISCUSSION THREAD

**Discussion by jaspermowatt** [opengisch/QField#2893] [Q&A]
Is there a way to use the advanced bee farm as a template project that can be bought into qgis to modify to suit my needs?

I like the way it’s set out but I struggle to replicate, especially with the photo capture etc.

Cheers

jasper

**Comment by signedav** [comment]
I think the easiest way is to get it from the resources folder in the repo https://github.com/opengisch/QField/tree/master/resources/sample_projects
